docs/d/hypervisor_manager.html.markdown

```diff
@@ -0,0 +1,53 @@
+---
+layout: "oneview"
+page_title: "Oneview: hypervisor_manager"
+sidebar_current: "docs-hypervisor_manager"
+description: |-
+ Gets information about an existing hypervisor_manager.
+---
+
+# oneview\_hypervisor\_manager
+
+Use this data source to access the attributes of a hypervisor manager.
+
+## Example Usage
+
+```hcl
+data "oneview_hypervisor_manager" "test" {
+ name = "Test hypervisor manager"
+}
+
+output "oneview_hypervisor_manager_value" {
+ value = "${data.oneview_hypervisor_manager.test.uri}"
+}
+```
+
+## Argument Reference
+
+* `uri` - (Required) The name of the hypervisor manager.
+
+## Attributes Reference
+
+* `fabric_type` - The supported Fibre Channel access method.
```

Comment:
The attributes specified here belong to some other resource, add the attributes and its respective description of hypervisor manager resource.

Reply:
Done